syntax = "proto3";
import "google/protobuf/any.proto";

package fedvision.framework;

message Task {
  string job_id = 1;
  string task_id = 2;
  string task_type = 3;
  google.protobuf.Any task = 4;
  string assignee = 5;
}

